### PR TITLE
Fixed docstrings in the util/datastore module

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

File: util/data_store/graph_store.py [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
-        """TBD"""
+        """List all the files in the Graph database."""
+        # TODO: not implemented
         return None
 
     def write_json_file(self, filename, contents):
-        """Write JSON file into data source"""
+        """Write JSON file into data source."""
+        # TODO: not implemented
+        pass
```
Comment (jpopelka) [inline util/data_store/graph_store.py]: These methods should rather `raise NotImplementedError`, but I understand it's not part of this PR.

Comment (tisnik) [reply util/data_store/graph_store.py]: yeah, I did not want to change the behaviour (well OTOH this function does not seem to be called anyway ;)
